content/en/docs/tasks/administer-cluster/migrating-from-dockershim/migrate-dockershim-dockerd.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Migrate Docker Engine nodes from dockershim to cri-dockerd"
+weight: 9
+content_type: task 
+---
+
+{{% thirdparty-content %}}
+
+This page shows you how to migrate your Docker Engine nodes to use `cri-dockerd`
+instead of dockershim. Follow these steps if your clusters run Kubernetes 1.23
+or earlier and you want to continue using Docker as your container runtime after
+you upgrade to Kubernetes 1.24 and later.
```

Comment:
These steps are appropriate for people who want to move ahead of v1.24, but they're also appropriate for considerate folk who are moving off the deprecated dockershim just because it's deprecated.